e2e/nomostest/predicates.go

```diff
@@ -54,7 +55,8 @@ var ErrObjectNotFound = errors.New("object not found")
 // WrongTypeErr reports that the passed type was not equivalent to the wanted
 // type.
 func WrongTypeErr(got, want interface{}) error {
-	return fmt.Errorf("%w: got %T, want %T", ErrWrongType, got, want)
```

Comment:
Do we still have a need for `ErrWrongType`? It looks like at this point it's basically just used as a string const

Reply:
Probably not... Next time ;)